build/virt-v2v/Containerfile-upstream

```diff
@@ -9,12 +9,19 @@ RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor ./cmd/
 RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter ./cmd/image-converter/image-converter.go
 RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-wrapper ./cmd/virt-v2v/entrypoint.go
 
+FROM registry.access.redhat.com/ubi9:9.5-1739449058 as virtio-win
+RUN curl -kL https://download.devel.redhat.com/brewroot/vol/rhel-7/packages/virtio-win/1.9.4/2.el7.2/noarch/virtio-win-1.9.4-2.el7.2.noarch.rpm -o virtio-win.rpm
+RUN dnf -y install ./virtio-win.rpm
```

Comment:
This is okay for development but for the proper build we will have it from the repos directly https://github.com/kubev2v/forklift/pull/1643